source/_integrations/aquacell.markdown

```diff
@@ -14,26 +14,90 @@ ha_platforms:
 ha_integration_type: device
 ---
 
-AquaCell is a water-softening device. The **AquaCell** {% term integration %} allows you to monitor your AquaCell device in Home Assistant.
+AquaCell is a water-softening device. The [AquaCell](https://www.aquacell-waterontharder.nl/) {% term integration %} allows you to monitor your AquaCell device in Home Assistant.
 You will need your Aquacell account information as used in the **AquaCell** app.
 
-This integration also supports **Harvey** softeners.
+This integration also supports [Harvey](https://www.harveywatersofteners.co.uk/) softeners.
```

Comment:
Should this be added as an virtual integration instead?

Reply:
Yes, I plan to make a PR for that soon. But it seemed right to also mention it here.